student/pom.xml

```diff
@@ -24,7 +24,7 @@
 	<properties>
 		<progman-client.version>3.0.1</progman-client.version>
 		<sb11-mna-client.version>3.0.0</sb11-mna-client.version>
-		<sb11-shared-code.version>3.0.0</sb11-shared-code.version>
+		<sb11-shared-code.version>3.0.1-SNAPSHOT</sb11-shared-code.version>
```

Comment:
Looks like we should release the shared code and then bump these versions to release.  you mentioned it on @mtrupkin's review as well.  I can help you do this or you can follow the steps [here](https://confluence.fairwaytech.com/display/SBAC1719/Releasing)

snapshot dependency references should no longer be committed to develop.